src/victory-util/immutable.js

```diff
@@ -0,0 +1,33 @@
+export default {
+  IMMUTABLE_ITERABLE: "@@__IMMUTABLE_ITERABLE__@@",
+  IMMUTABLE_RECORD: "@@__IMMUTABLE_RECORD__@@",
+  IMMUTABLE_LIST: "@@__IMMUTABLE_LIST__@@",
+  IMMUTABLE_MAP: "@@__IMMUTABLE_MAP__@@",
+
+  isImmutable(x) {
+    return this.isIterable(x) || this.isRecord(x);
+  },
+
+  isIterable(x) {
+    return !!(x && x[this.IMMUTABLE_ITERABLE]);
+  },
+
+  isRecord(x) {
+    return !!(x && x[this.IMMUTABLE_RECORD]);
+  },
+
+  isList(x) {
+    return !!(x && x[this.IMMUTABLE_LIST]);
+  },
+
+  isMap(x) {
+    return !!(x && x[this.IMMUTABLE_MAP]);
+  },
+
+  shallowToJS(x) {
+    return this.isIterable(x) ? x.reduce((prev, curr, key) => {
+      prev[key] = curr;
+      return prev;
+    }, this.isList(x) ? [] : {}) : x;
+  }
+};
```

Comment:
these methods should work for `immutable@v3.8.x` as well as for `v4.0.x` when that is released.